src/ipahealthcheck/system/filesystemspace.py

```diff
@@ -2,13 +2,29 @@
 # Copyright (C) 2019 FreeIPA Contributors see COPYING for license
 #
 
+import os
 import shutil
 
 from ipahealthcheck.system.plugin import SystemPlugin, registry
 from ipahealthcheck.core.plugin import duration, Result
 from ipahealthcheck.core import constants
 
 
+def in_container():
+    """Determine if we're running in a container."""
+    with open('/proc/1/sched', 'r') as sched:
+        data = sched.readline()
+
+    checks = [
+        data.split()[0] not in ('systemd', 'init',),
```

Comment:
I'm not sure whether that would produce a false negative in a systemd container (which I know nothing about atm).

Reply:
I don't know if IPA runs in it, that would I think be our driving interest.

Reply:
Indeed. And that line takes care of both docker and podman/runc too.